pkgs/sdk_triage_bot/lib/src/github.dart

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: avoid_dynamic_calls
+
+import 'package:github/github.dart';
+import 'package:graphql/client.dart';
+
+import 'common.dart';
+
+abstract class GithubService {
+  Future<List<String>> getAllLabels(RepositorySlug repoSlug);
+
+  Future<Issue> getIssue(RepositorySlug sdkSlug, int issueNumber);
```

Comment:
[nit] https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get

Either dropping the "get" or replacing it with "fetch" would be an improvement.

Reply:
Will do -